App/Inreach.php

```diff
@@ -393,6 +407,14 @@ function build_geojson() {
 					
 						//We have data														
 						if(sizeof($extended_data)) {
+							//fuzz the coordinates that are being displayed in the table
```

Comment:
Please see my previous comment above including this in this [switch](https://github.com/OpenGIS/inreach-mapshare/blob/83181e84ac68d7d30b27813c02b32de97aba9b8f/App/Inreach.php#L295), that is for processing output by Key.